solr/core/src/java/org/apache/solr/handler/admin/CollectionsHandler.java

```diff
@@ -1791,14 +1794,17 @@ public Map<String, Object> execute(
     REPLACENODE_OP(
         REPLACENODE,
         (req, rsp, h) -> {
-          return copy(
-              req.getParams(),
-              null,
-              "source", // legacy
```

Comment:
[-1] I'm fine with dropping the deprecated old parameter names in this PR, but if we drop them here, we should also drop the code that checks for them in [ReplaceNodeCmd](https://github.com/apache/solr/blob/c99af207c761ec34812ef1cc3054eb2804b7448b/solr/core/src/java/org/apache/solr/cloud/api/collections/ReplaceNodeCmd.java#L67)

Reply:
Sure